pytorch_lightning/core/lightning.py

```diff
@@ -550,7 +550,7 @@ def __check_allowed(v: Any, name: str, value: Any) -> None:
     def __to_tensor(self, value: numbers.Number) -> torch.Tensor:
         return torch.tensor(value, device=self.device)
 
-    def log_grad_norm(self, grad_norm_dict: Dict[str, torch.Tensor]) -> None:
+    def log_grad_norm(self, grad_norm_dict: Dict[str, float]) -> None:
```

Comment:
This doesnt seem right, i'll have to check :) reminder to myself.

Reply:
the origin of the float values is the [grad_norm](https://github.com/PyTorchLightning/pytorch-lightning/blob/912fd3113143915d3d56f34ec44888911e212926/pytorch_lightning/utilities/grads.py#L51) utility function. This function is called in the loops and then the dict of floats propagates to the LightningModule where I had too change the type